packages/a11y/docs/VisuallyHidden.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+import { withDesign } from "storybook-addon-designs"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+
+import { VisuallyHidden } from ".."
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.a11y}/Visually Hidden`,
```

Comment:
There's a 'helpers' section where this would make more sense (rather than inside components)
![image](https://user-images.githubusercontent.com/1811583/140011294-dacbb5d7-afb8-46f9-863d-879f4664a125.png)

Reply:
🤔 IMO helpers are packages that don't necessarily have UI. I'm only questioning this because I also have a Pr that will belong in the a11y package but (I don't think) is a helper: https://github.com/cultureamp/kaizen-design-system/pull/2201

Reply:
Makes me wonder whether they do belong in the same package then, or if we want a general `utils` package instead.

Reply:
Don't have strong opinions on this, but seeing there now will be 2 components related to A11y (`SkipLink` & `VisuallyHidden`) it feels like an OK grouping to me.